front/app/components/PostShowComponents/Comments/ChildCommentForm.tsx

```diff
@@ -147,8 +152,8 @@ class ChildCommentForm extends PureComponent<Props & InjectedIntlProps, State> {
         )
         .subscribe(({ eventValue }) => {
           const { authorFirstName, authorLastName, authorSlug } = eventValue;
-          const inputValue = `@[${authorFirstName} ${authorLastName}](${authorSlug}) `;
-          this.setState({ inputValue, focused: true });
+          const tag = `@[${authorFirstName} ${authorLastName}](${authorSlug}) `;
+          this.setState({ tag, inputValue: tag, focused: true });
```

Comment:
is this double definition (tag, inputValue) really needed?

Reply:
Yes, at a later time they are different. But I was just thinking that it's possible to remove this tag and then the submit button would be blocked even if you typed something I think. I think this solution is likely not okay in any case, so thanks for your comment